### DISCUSSION TITLE
Still under development?

### DISCUSSION THREAD

**Discussion by declancarrollcb** [BugDiver/gauge-ts#133] [General]
Hey,
I'm just checking if this is still under active development. If so what would be needed to get it to a 1.0.0 release and get it moved out of experimental status?

I'm happy to help out anyway I can.

**Comment by bugdiver** [comment]
@declancarrollcb This supports all the Gauge features. There was no specific reason to not make it 1.0.0
Is there a place in doc (or any other place) where you see it marked as Experimental?

**Reply by declancarrollcb** [reply]
I could have been getting it confused with the Taiko bindings for typescript in regards to the experimental support. Will this repo be getting moved https://github.com/getgauge/gauge-js/issues/187, or will it be staying here? Either way, thank you for the work you've done it's really great to use.

**Reply by bugdiver** [reply]
@declancarrollcb This will stay here.
